services/core-web/src/components/common/DocumentTable.tsx

```diff
@@ -80,6 +57,7 @@ export const DocumentTable: FC<DocumentTableProps> = ({
   const [rowSelection, setRowSelection] = useState([]);
   const [isCompressionModal, setCompressionModal] = useState(false);
   const [isCompressionInProgress, setCompressionInProgress] = useState(false);
+  const [documentTypeCode, setDocumentTypeCode] = useState("");
```

Comment:
Are we calling setDocumentTypeCode anywhere? I don't think this will ever be anything other than "". I think it got half-implemented, and then I mentioned that the initial value was going to be undefined a lot of the time, and it didn't get fully removed.